### PR TITLE
 #48 | Add contact page with form

### LINKED ISSUE #48: [Blocked by client] Contact page
Create a contact page with a form to contact

### PR DESCRIPTION
**Pull request by juanjoseortega-aircury**

### CODE REVIEW

File: assets/custom.css
Comment (cristobaldiaz-aircury) [inline assets/custom.css]: This could be moved to another file. Custom is growing rapidly

File: contact.md
Comment (cristobaldiaz-aircury) [inline contact.md]: This comment should be implemented or removed

File: assets/js/contact_form.js
Comment (cristobaldiaz-aircury) [inline assets/js/contact_form.js:2]: It should be `https`

File: assets/custom.css
Comment (cristobaldiaz-aircury) [inline assets/custom.css]: The colour codes are in both capital and non capital letters. The same format should be used for all of them.

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
The layout for the name and email doesn't look good on mobiles, you cannot see all the text:
image

File: assets/js/contact_form.js
Comment (juanjoseortega-aircury) [reply assets/js/contact_form.js:2]: If i use https doesn't redirect in localhost correctly, but in this case using http at production it will be redirected to https

**Review by cristobaldiaz-aircury — APPROVED** [review]
Pending to ask client for a production token.

Unrelated but worth mentioning, we are not using master but preview. We should review that.

**Review by cristobaldiaz-aircury — CHANGES_REQUESTED** [review]
Blocked by client. The form in another website is suffering from spam flooding, even with a captcha solution.

**Pull request closed without merging**
